cookbooks/aws-parallelcluster-environment/recipes/config/network_interfaces.rb

```diff
@@ -16,7 +16,13 @@
 return if on_docker?
 
 def network_card_index(mac, token)
+  # This IMDS call is not available on single NIC instance, therefore fallback to 0
```

Comment:
It would be good to link in a comment the public documentation where this behavior is described.